test/__tests__/data/mp_output.json

```diff
@@ -2033,5 +2033,103 @@
       "files": {},
       "userId": "anonId01"
     }
+  ],
+  [
+    {
+      "version": "1",
+      "type": "REST",
+      "method": "POST",
+      "endpoint": "https://api.mixpanel.com/import/",
+      "headers": {
+        "Authorization": "Basic OTQzMmYxMWY3MGY4Y2UzODZmNTExMGM4YzkyNGIzZWM0ZjgyNTI1Njo="
+      },
+      "params": {
+        "data": {
+          "event": "Application Installed",
+          "properties": {
+            "$app_build_number": "4",
+            "$app_version_string": "1.0",
+            "$bluetooth_enabled": true,
+            "$carrier": "T-Mobile",
+            "$device": "emu64a",
+            "$insert_id": "168cf720-6227-4b56-a98e-c49bdc7279e9",
+            "$manufacturer": "Google",
+            "$model": "sdk_gphone64_arm64",
+            "$os": "Android",
+            "$os_version": "12",
+            "$screen_dpi": 560,
+            "$screen_height": 2984,
+            "$screen_width": 1440,
+            "$session_id": "1662363980",
+            "$wifi": true,
+            "anonymousId": "39da706ec83d0e90",
+            "build": 4,
+            "distinct_id": "39da706ec83d0e90",
+            "mp_device_model": "sdk_gphone64_arm64",
+            "mp_lib": "com.rudderstack.android.sdk.core",
+            "time": null,
```

Comment:
shouldn't `time` load with originalTimestamp field value with the fallback logic introduced now ?

Reply:
The field that you pointed here, is something else
The fix made as part of the bugsnag issue was to populate the field `$time` in the `params.data.$append[]`
The field `time` pointed here is making use of `message.timestamp` itself. But in order to populate this field, we have to make changes elsewhere. If it is confirmed to make changes, I can make them

Reply:
There are other parts of the code where in `timestamp` is being used. We need to also change it but for now. I can make the changes for only `track` call alone.

@krishna2020 Can you confirm if `time` also needs to be change such that when `timestamp` is not proper, we need to use `originalTimestamp` ?